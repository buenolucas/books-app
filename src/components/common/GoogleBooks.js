import React from 'react';
import {Image} from 'react-native';

const GoogleBooks = () => (
  <Image source={require('../../assets/img/logo.png')} />
);

export default GoogleBooks;
