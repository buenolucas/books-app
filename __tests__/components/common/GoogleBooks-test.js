import React from 'react';
import {shallow} from 'enzyme';
import GoogleBooks from '../../../src/components/common/GoogleBooks';

describe('GoogleBooks', () => {
  describe('Rendering', () => {
    it('should match to snapshot', () => {
      const component = shallow(<GoogleBooks />);
      expect(component).toMatchSnapshot();
    });
  });
});
