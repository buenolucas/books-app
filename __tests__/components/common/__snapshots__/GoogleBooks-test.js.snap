// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleBooks Rendering should match to snapshot 1`] = `
<Image
  source={
    Object {
      "testUri": "../../../src/assets/img/logo.png",
    }
  }
/>
`;
